__tests__/app.test.js

```diff
@@ -327,3 +327,33 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+
+describe("GET /api/users", () => {
+  test("should return all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("should return error 400 for an invalid route ", () => {
+    return request(app)
+      .get("/api/usernames")
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toEqual("Bad Request");
+      });
+  });
```

Comment:
This test is not required and is a bit muddled.  Any nonexistent route on the app/api should be a 404 status. This should be checked on an app level which I imagine/hope you already have if you want a custom message for this. If not then Express will send beck the 404 for you